### DISCUSSION TITLE
Anyway to let Steam Page handle the authentication flow?

### DISCUSSION THREAD

**Discussion by 1MochaChan1** [ValvePython/steam#430] [Q&A] [answered]
Hey, I am working on a website where the user may need to sign in to his Steam account so they can see the games they own.
According to SteamWorks documentation here in the [Web Browser based authentication with OpenID](https://partner.steamgames.com/doc/features/auth) section, they have mentioned we can use their OpenID 2.0 implementation to do so.

So, is it possible for us to redirect the user to the Steam Page for authentication and then get the `steam_id` once they've successfully logged in?

I am using the following: 
`Python 3.11`
`Flask 2.2.2`
`steam 1.4.4`

**Comment by rossengeorgiev** [comment, marked as answer]
If you are making website you can let users login with OpenID. You will get their Steam ID, and you can then use the Steam WebAPI to request information, like the games they own as long as they have their profile privacy set to public.

You don't need `steam` package for any of that.

**Reply by 1MochaChan1** [reply]
Thanks a lot! I am new to Web Development so I wasn't sure what to use. Appreciate your quick response.
